src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -292,6 +292,10 @@
         - name: --node-resource-group
           type: string
           short-summary: The node resource group is the resource group where all customer's resources will be created in, such as virtual machines.
+        - name: --nrg-restriction
```

Comment:
@FumingZhang curious why there is a `22` character limit on the option lengths:
https://dev.azure.com/azclitools/public/_build/results?buildId=25902&view=logs&j=ccdac6d3-465a-5ada-8819-d5b4a6b3a631&t=ffdbba7d-e523-5908-c7bf-0703bafb81b0&l=472

Would have preferred to use `--nrg-lockdown-restriction-level` which was approved in the API review. However, due to the character limit I had to switch it to `--nrg-restriction` as seen currently in code

Reply:
Hey @charliedmcb, you can use the option names designed and approved before. To bypass such CI checks, just update file `src/aks-preview/linter_exclusions.yml` and add an entry about this new option.

Reply:
Ok, thanks! :)

Reply:
I'll look at updating this tomorrow, and ping you for a final review when ready

Reply:
Updated